.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js: "8"
+stages: 
+    - name: unit_tests
+    
+jobs:
+    include:
+        - stage: unit_tests
+          env: CHECK=UNIT_TESTS
+          script: 
+            - "npm install mocha chai"
+            - "./node_modules/.bin/mocha ./testsRestApi.js"
+            - "./node_modules/.bin/mocha ./testsSysLogContent.js"
```

Comment:
Instead of specifying the tests individually, use `npm run test`, this will automatically be resolved into testing restapi and syslogs

Reply:
It has been updated to just npm run test.